src/docs/metrics/defi/index.md

```diff
@@ -42,13 +50,11 @@ Dollars/Amount of coins
 ---
 
 ## Available Assets
-
-Available for `ethereum`.
+Available for `ethereum` and `usd`.
```

Comment:
What is the `usd` asset? Should this be an asset/currency pair?

Reply:
Fixed